cpp/monitoring/gcm/exporter.cc

```diff
@@ -0,0 +1,264 @@
+#include "monitoring/gcm/exporter.h"
+
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+#include <sstream>
+
+#include "monitoring/metric.h"
+#include "monitoring/registry.h"
+#include "net/url.h"
+#include "util/json_wrapper.h"
+#include "util/sync_task.h"
+
+DEFINE_string(google_compute_monitoring_push_url, "",
+              "GCM custom timeseries write URL.");
+DEFINE_string(google_compute_monitoring_metrics_url, "",
+              "GCM custom timeseries metrics URL.");
+DEFINE_int32(google_compute_monitoring_push_interval_seconds, 5,
```

Comment:
The 5s interval seems very short!  Is that a recommendation from GCM?  I assume this consumes some kind of quota faster than say a 30s interval would do.